x/mongo/driver/topology/tls_connection_source_1_16.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) MongoDB, Inc. 2017-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// +build !go1.17
```

Comment:
Consider adding the new build tag format as well.

E.g.
```go
//go:build !go1.17
```

Reply:
Added.